shard.yml

```diff
@@ -16,8 +16,6 @@ dependencies:
 development_dependencies:
   minitest:
     github: ysbaddaden/minitest.cr
-    version: ~> 0.5.0
```

Comment:
Removing these seems out of place with the rest of the pull request. Can you give some explanation?